x-pack/plugins/secops/public/components/timeline/body/column_headers/range_picker/ranges.ts

```diff
@@ -4,8 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-/** Date ranges applicable to the MiniMap */
-export type Range = '1 Day' | '1 Week' | '1 Month' | '1 Year';
+import * as i18n from './translations';
 
 /** Enables runtime enumeration of valid `Range`s */
-export const Ranges: Range[] = ['1 Day', '1 Week', '1 Month', '1 Year'];
```

Comment:
Not that I know of since it's user facing translatable code that you don't know what the value is going to be until after its plucked from a translation file.